src/locales/strings.js

```diff
@@ -0,0 +1,61 @@
+ // TODO(sjbarag): read from JS module generated from JSON files, probably.
+const translations = {
+  en: {
+    "hello": {
+      title: "hello, {name}",
+      note: "Says hi to a person.",
+    },
+    "usePersonalDuckAddr": {
+      title: "Use {email}",
+      note: "Shown when a user can choose their personal @duck.com address.",
+    },
+    "blockEmailTrackers": {
+      title: "Block email trackers",
+      note: "Shown when a user can choose their personal @duck.com address on native platforms.",
+    }
+  },
+  xa: {
+    "hello": {
+      title: "h33llllo, {name}",
+      note: "Says hi to a person.",
+    },
+    "usePersonalDuckAddr": {
+      title: "Ü55££ {email}",
+      note: "Shown when a user can choose their personal @duck.com address.",
+    },
+    "blockEmailTrackers": {
+      title: "Bl000ck €m@@@i1il1l träáåck33rr55",
+      note: "Shown when a user can choose their personal @duck.com address on native platforms.",
+    }
+  },
+};
+
+export function getTranslator(locale) {
+  let library = translations[locale];
+  if (library == null) {
+    console.warn(`Received unsupported locale '${locale}'. Falling back to 'en'.`);
+    library = translations.en;
+  }
+
+  return function t(id, opts) {
+    return translateImpl(library, id, opts);
+  }
+};
```

Comment:
This is a wrapper around `translateImpl`, mostly because I didn't want to have to pass the locale around all the time. It feels reasonable, but if you've got a better idea let me know!